R/tab-book.R

```diff
@@ -121,6 +121,35 @@ extToContentType <- function(ext) {
     return(mapping[[ext]])
 }
 
+# Possibly went a little overboard allowing different filter options in tabbook
+# extract out the logic here
+standardize_tabbok_filter <- function(dataset, filter) {
```

Comment:
```suggestion
standardize_tabbook_filter <- function(dataset, filter) {
```
Great idea extracting this!
Definitely the right way to go but to get the real benefit means some mildly tedious test rewriting (it's still the right thing to do, since we rushed the first version of it out):
all of the test cases should be taken out of `tabBook` — probably wrap them in `toJSON` and just move those json bits over to new unit tests.